app/controllers/competencies_controller.rb

```diff
@@ -1,4 +1,5 @@
 class CompetenciesController < ApplicationController
+  respond_to :html, :json
```

Comment:
I think this line here will tell `respond_with @competency` to render it in both html and json.